security.rst

```diff
@@ -1838,30 +1837,28 @@ To enable logging out, activate the  ``logout`` config parameter under your fire
             $mainFirewall = $security->firewall('main');
             // ...
             $mainFirewall->logout()
-                // the argument can be either a route name or a path
-                ->path('app_logout')
+                ->path('/logout')
 
                 // where to redirect after logout
                 // ->target('app_any_route')
             ;
         };
 
-``path`` can be either a route name, or a URI.
+Symfony will then un-authenticate users navigating to the configured ``path``,
+and redirect them to the configured ``target``.
 
-Since Symfony 6.4, you can leverage the
-:class:`Symfony\\Bundle\\SecurityBundle\\Routing\\LogoutRouteLoader`
-to automatically register a route for every logout URI you configured.
-If your project uses :ref:`Symfony Flex <symfony-flex>`,
-this is already done for you.
-Otherwise, you need to import the ``security.route_loader.logout`` service
-as a routing resource:
+If your project uses :ref:`Symfony Flex <symfony-flex>`, you're all set:
+the :class:`Symfony\\Bundle\\SecurityBundle\\Routing\\LogoutRouteLoader` will
+automatically register a route for every logout URI you configured!
+Otherwise, you can enable this behavior yourself by importing the
+``security.route_loader.logout`` service as a routing resource:
```

Comment:
The logout route loader is a bit of an implementation detail that I would not mention in the docs (it only makes things complicated).

```suggestion
If your project does not use :ref:`Symfony Flex <symfony-flex>`, make sure
you have imported the logout route loader in your routes:
```

Reply:
I can avoid mentioning the `LogoutRouteLoader` but wouldn’t it be better to explain what it does?

Reply:
Let's see what other documentors think about this, but I would say: no.

Our documentation is often focused on the non-seasoned developer, especially the main guides like this one. Mentioning advanced unrelated topics (i.e. route loaders in a security guide) can be very confusing for people that want to learn how to secure their Symfony applications.
For seasoned devs that do understand all the concepts and want to know what is generating this logout URL, the shown routing config should give enough hints to find this class based on the service ID.

As we have rather low resources on documentation, we favor not spending much resources on helping seasoned developers that know how to find their way in the Symfony source.

Reply:
Let's go with Wouters' decision here 👍